lib/core/services/alfresco-api.service.ts

```diff
@@ -79,15 +109,21 @@ export class AlfrescoApiService {
             domainPrefix : this.appConfig.get<string>(AppConfigValues.STORAGE_PREFIX),
             oauth2: oauth
         });
+    }
 
-        if (this.alfrescoApi && this.isDifferentConfig(this.lastConfig, config)) {
-            this.lastConfig = config;
-            this.alfrescoApi.setConfig(config);
+    protected initAlfrescoApi() {
+        this.getCurrentAppConfig();
+        this.initAlfrescoApiWithLastConfig();
+    }
+
+    private initAlfrescoApiWithLastConfig() {
+        if (this.alfrescoApi && this.isDifferentConfig(this.lastConfig, this.currentAppConfig)) {
+            this.lastConfig = this.currentAppConfig;
```

Comment:
the method name says initAlfrescoApiWithLastConfig but in practice is initialising with the current config because you are doing  `this.lastConfig = this.currentAppConfig;`